cove_bods/templates/cove_bods/additional_checks_table.html

```diff
@@ -235,7 +235,7 @@
         {% elif additional_check.type == 'statement_is_component_but_is_after_use_in_component_statement_id' and additional_check.statement_type == 'person'  %}
             <tr>
                 <td>
-                    {% blocktrans %}This Person Statement is a component (isComponent) and should appear <strong>before</strong> the primary Ownership-or-control Statement that references it (from componentStatementIDs).{%endblocktrans%}
+                    {% blocktrans %}<code>Person statement</code> not in correct order. As a component (<code>isComponent</code> 'true'), it must appear <strong>before</strong> the primary Ownership-or-control statement that references it (from <code>componentStatementIDs</code></).{% endblocktrans %}
```

Comment:
```suggestion
                    {% blocktrans %}<code>Person statement</code> not in correct position. As a component (<code>isComponent</code> 'true'), it must appear <strong>before</strong> the primary Ownership-or-control statement that references it (from <code>componentStatementIDs</code></).{% endblocktrans %}
```

this is me being nitpicky but the collection of statements isn't in the right order, the statement itself is in the wrong position?